src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -2499,6 +2499,37 @@ private static Stream GetStreamFromString(string s)
         stream.Position = 0;
         return stream;
     }
+
+    [ActiveIssue(12799)]
```

Comment:
To cover the XmlTypeMapping scenaior, I need use SoapAttributes. I can add a separate test to cover the SaopAttribute.